source/Player.hx

```diff
@@ -0,0 +1,215 @@
+		if (isRunning)
+		{
+			xMaxSpeed = runSpeed;
+		}
+		else
+		{
+			xMaxSpeed = walkSpeed;
+		}
+		
+		// If horizontalMove is -1, the Player should move left.
+		if (horizontalMove == -1)
+		{
+			if (velocity.x > 0)
+			{
+				acceleration.x = -xSlowdown + -xAccel;
+			}
+			else
+			{
+				acceleration.x = -xAccel;
+			}
+		}
+		
+		// If horizontalMove is 1, the Player should move right.
+		else if (horizontalMove == 1)
+		{
+			if (velocity.x < 0)
+			{
+				acceleration.x = xSlowdown + xAccel;
+			}
+			else
+			{
+				acceleration.x = xAccel;
+			}
+		}
+		
+		// Slow down if no direction held
+		else if (horizontalMove == 0)
+		{
+			if (velocity.x > 0)
+			{
+				if (velocity.x <= xSlowdown * elapsed)
+				{
+					velocity.x = 0;
+					acceleration.x = 0;
+				}
+				else
+				{
+					acceleration.x = -xSlowdown;
+				}
+			}
+			else if (velocity.x < 0)
+			{
+				if (velocity.x >= -xSlowdown * elapsed)
+				{
+					velocity.x = 0;
+					acceleration.x = 0;
+				}
+				else
+				{
+					acceleration.x = xSlowdown;
+				}
+			}
+		}
+		#if debug // Only compile this code into a debug version of the game.
+		
+		// Display an error message in the console if an invalid horizontalMove
+		// 	value is detected.
+		else
+		{
+			trace("ERROR: An invalid value for horizontalMove (" + 
+				horizontalMove + ") was passed into groundMovement()");
+		}
+		
+		#end // End of the conditional compilation section.
+	}
+	
+	private function airMovement(isRunning:Bool, horizontalMove:Int, elapsed:Float):Void
+	{
+	// Change max speed if the player is running
+		if (isRunning)
+		{
+			xMaxSpeed = runSpeed;
+		}
+		else
+		{
+			xMaxSpeed = walkSpeed;
+		}
+		
+		// If horizontalMove is -1, the Player should move left.
+		if (horizontalMove == -1)
+		{
+				acceleration.x = -xAccel;
+		}
+		
+		// If horizontalMove is 1, the Player should move right.
+		else if (horizontalMove == 1)
+		{
+			acceleration.x = xAccel;
+		}
+		
+		// Stop horizontal acceleration if no direction held
+		else if (horizontalMove == 0)
+		{
+			acceleration.x = 0;
+		}
+		
+		#if debug // Only compile this code into a debug version of the game.
+		
+		// Display an error message in the console if an invalid horizontalMove
+		// 	value is detected.
+		else
+		{
+			trace("ERROR: An invalid value for horizontalMove (" + 
+				horizontalMove + ") was passed into airMovement()");
+		}
+		
+		#end // End of the conditional compilation section.
+	}
+	
+	public override function update(elapsed:Float):Void
+	{
+		// Determine if running
+		var isRunning:Bool = FlxG.keys.anyPressed([FlxKey.Z, FlxKey.PERIOD]);
+		
+		// Determine direction of movement
+		var horizontalMove:Int = 0;
+		
+		if (FlxG.keys.anyPressed([FlxKey.LEFT, FlxKey.A]))
+		{
+			horizontalMove--;
+		}
+		if (FlxG.keys.anyPressed([FlxKey.RIGHT, FlxKey.D]))
+		{
+			horizontalMove++;
+		}
+		
+		// Determine if attempted to jump
+		var attemptedJump:Bool = FlxG.keys.anyJustPressed([FlxKey.X, FlxKey.BACKSLASH]);
```

Comment:
On your commit comment, you said that the player can jump with the '/' key. But in the code you reference the '\' key instead. I would recommend changing the key to the '/' as it is next to the '.' key for going into "maximum overdrive".